client/js/components/user/DpOrganisationList/DpOrganisationList.vue

```diff
@@ -31,14 +31,15 @@
           class="o-list o-list--card u-mb"
           data-cy="pendingOrganisationList">
           <dp-organisation-list-item
```

Comment:
Not sure why we have to differente list items. Do we need the prop `:additional-field-options="additionalFieldOptions"` here, too?

Reply:
I'm not sure if it's needed either, but it seemed safer to add it, and I don't think it does any harm.

Reply:
Did you push the changes? I don't see it added

Reply:
Ah, I think I misunderstood. Your concern was that it _should_ be added? I remember adding it, but you're right, it's definitely not there. :smile: I'll add it